episodes/01-welcome.md

```diff
@@ -268,6 +268,7 @@ workshops specifically.
 [form-coc]: https://goo.gl/forms/KoUfO53Za3apOuOK2
 [lesson-dev]: https://carpentries-incubator.github.io/study-groups/
```

Comment:
is this now irrelevant and we should remove it?

Reply:
I did not want to remove the link to the Study Groups on my own (there is also the Curriculum Development Handbook which may be superseded by the new CLDT curriculum but I find a useful resource still so did not want to remove references to that either) - think @tobyhodges should weigh in on this one and make the final call.

Reply:
fine to remove it, I think. The Study Groups curriculum itself doesn't add anything not provided by the Lesson Development Training curriculum.